syncd/syncd.cpp

```diff
@@ -351,6 +529,13 @@ void internal_syncd_get_send(
                 attr_count,
                 attr_list,
                 false);
+
+        if (isInitViewMode())
+        {
+            // all oid values here are VID's
+
+            snoop_get_response(object_type, str_object_id, attr_count, attr_list);
+        }
```

Comment:
why is that the snoop op is only applied in the init view mode? The snoop op is trying to capture the asic state which are created by default in the sai library, so that we can use them for later comparison purpose.

for example, we first get 8 queues from asic, and remove 2 of them. In the asic view, we should have remaining 6. the snoop should be performed even during non init view phase.